diff --git a/OVSAgentAutoStart.py b/OVSAgentAutoStart.py
index 88fa18c..794a363 100755
--- a/OVSAgentAutoStart.py
+++ b/OVSAgentAutoStart.py
@@ -111,8 +111,6 @@ def prepare_cluster_heartbeat():
 JOBS = [
     ["command", join(dirname(__file__), "utils/upgrade_agent.py")],
     ["function", "get_agent_version", ()],
-    ["function", "prepare_cluster_root", ()],
-    ["function", "prepare_cluster_heartbeat", ()],
     ["command", join(dirname(__file__), "utils/upgrade.py")],
     ["command_bg", join(dirname(__file__), "OVSRemasterServer.py")],
     ["command_bg", join(dirname(__file__), "OVSMonitorServer.py")],
