*** OVSServices.py	2011-06-06 12:31:23.279919998 -0700
--- /tmp/OVSServices.py	2011-06-06 12:32:13.275919997 -0700
***************
*** 68,73 ****
--- 68,75 ----
  import OVSVMXParser                 #pylint:    disable-msg=W0611
  import OVSVMDKParser                #pylint:    disable-msg=W0611
  
+ import OvmDispatcher
+ 
  from OVSWrappers import D
  from OVSCommons import exposed
  
***************
*** 113,118 ****
--- 115,121 ----
          self.sleep = sleep
  
          self._load_modules()
+         OvmDispatcher.InitOvmDispacther()
  
  #        #xenapi
  #        import xen.xend.XendAPI as XendAPI
